dome9CloudBots/bots/storage_account_disable_public_network_access.py

```diff
@@ -0,0 +1,49 @@
+# What it does: Restricts Storage account access to internal subnets only
+# Usage: AUTO: storage_account_disable_public_network_access <vnet resource group> <vnet> <subnet>

```

Comment:
No need to use 'AUTO: ' convention in usage and example. While supporting it we prefer the usage of the remediation in our system

Reply:
Amended